pom.xml

```diff
@@ -238,6 +238,7 @@
                     <encoding>UTF-8</encoding>
                     <consoleOutput>true</consoleOutput>
                     <failsOnError>true</failsOnError>
+                    <resourceIncludes>**/*.properties,**/*.jelly,**/*.html</resourceIncludes>
```

Comment:
Actually for html/jelly files it only does some general checks such as white spaces, new line at end of file, etc.